src/facet/selection/_parameters.py

```diff
@@ -253,20 +255,20 @@ def _values_to_expression(values: ParameterSet) -> Expression:
 
             return make_expression(values)
 
-        path_prefix = (
+        path_prefix_list: List[str] = (
```

Comment:
let's not add the type to the variable name: the type hint should be sufficient. rename back to `path_prefix`

Reply:
But then this will cause mypy redefinition error as `path_prefix` is already an argument to this method, and also has different type. To make mypy work where we can either: 
1. Cast `path_prefix` to List[str]
2. Introduce `path_prefix_list` as now that has desired type

Which one should I go with?

Reply:
Got it, all good!